### ISSUE TITLE
Curried setFieldValue by default?

### ISSUE THREAD

**Issue by cloud-walker** [jaredpalmer/formik#217]
I think this is pretty common, when you have a custom input, you have to use `setFieldValue` to first pass the name of the field, and many times I've found myself doing like this:

```javascript
<AwesomeInput onChange={setFieldValue('some_random_name')} value={values.some_random_name} />
```

**Comment by prichodko** [comment]
Hey @cloud-walker,

Can you explain, why
```javascript
<AwesomeInput name="some_random_name" onChange={handleChange} />
```
is not enough?

**Comment by cloud-walker** [comment]
Because in my case I have a custom input which has no real name, it's a Date input that accepts a date from the outside but from the inside renders 3 inputs (day, month and year).

Or a Downshift.js autocomplete, to give another example.

So, I can enforce the `name` prop for these components but I'm not sure is the right solution 🤔

**Comment by jaredpalmer** [comment]
Is it possible to accomplish this with non-breaking changes to setFieldValue?

**Comment by cloud-walker** [comment]
I think so, I can't talk for react-native tho.

Another "problem" is the currying function, in my codebase I use ramda, but maybe it's not a good fit for a library like formik 🤔

**Comment by jaredpalmer** [comment]
This has been released in 0.12-alpha. You can now call `handleChange('thing')`
